docs/core/resilience/http-resilience.md

```diff
@@ -0,0 +1,321 @@
+- Declares a `builder` (of type `IStandardHedgingHandlerBuilder`), which is used to configure the standard hedging resilience handler. There are extension methods to configure the standard hedging resilience handler.
+
+The `ExampleClient` is defined as follows:
+
+:::code source="snippets/http-resilience/ExampleClient.cs":::
+
+The preceding code:
+
+- Defines an `ExampleClient` type that has a constructor that accepts an <xref:System.Net.Http.HttpClient>.
+- Exposes a `GetCommentsAsync` method that sends a GET request to the `/comments` endpoint and returns the response.
+
+The `Comment` type is defined as follows:
+
+:::code source="snippets/http-resilience/Comment.cs":::
+
+### Standard hedging handler defaults
+
+The standard hedging uses a pool of circuit breakers to ensure that unhealthy endpoints aren't hedged against. By default, the selection from the pool is based on the URL authority (scheme + host + port).
+
+> [!TIP]
+> It's recommended that you configure the way the strategies are selected by calling `StandardHedgingHandlerBuilderExtensions.SelectPipelineByAuthority`.
+
+The preceding code adds the standard hedging handler to the <xref:Microsoft.Extensions.DependencyInjection.IHttpClientBuilder>. The default configuration chains five resilience strategies in the following order (from the outermost to the innermost):
+
+| Order | Strategy | Description |
+|--:|--|--|
+| **1** | Total request timeout | The total request timeout pipeline applies an overall timeout to the execution, ensuring that the request, including hedging attempts, doesn't exceed the configured limit. |
+| **2** | Hedging | The hedging strategy executes the requests against multiple endpoints in case the dependency is slow or returns a transient error. Routing is options, by default it just hedges the URL provided by the original <xref:System.Net.Http.HttpRequestMessage>. |
+| **3** | Rate limiter (per endpoint) | The rate limiter pipeline limits the maximum number of concurrent requests being sent to the dependency. |
+| **4** | Circuit breaker (per endpoint) | The circuit breaker blocks the execution if too many direct failures or timeouts are detected. |
+| **5** | Attempt timeout (per endpoint) | The attempt timeout pipeline limits each request attempt duration and throws if it's exceeded. |
+
+### Customize hedging handler route selection
+
+When using the standard hedging handler, you can customize the way the strategies are selected by calling various extensions on the `IRoutingStrategyBuilder` type. This can be useful for scenarios such as a/b testing, where you want to route a percentage of the requests to a different endpoint:
+
+```csharp
+// Hedging allows sending multiple concurrent requests.
+// The builder is of type, IHttpClientBuilder.
+builder.AddStandardHedgingHandler(static (IRoutingStrategyBuilder builder) =>
+{
+    builder.ConfigureOrderedGroups(static orderedGroup =>
+    {
+        orderedGroup.Groups.Add(new EndpointGroup()
```

Comment:
I suggest to add at least two groups. Having only one group means that hedging will never do secondary attempt. The maximum number of attempts is limited by number of groups, it's good to mention that.